tooling/template-base/components/Assets/heroImage.tsx

```diff
@@ -0,0 +1,4 @@
+
+export default HeroImage;
```

Comment:
Think for consistency we should export the full `<img>` element w/ `HeroImage` as its `src` instead of just exporting the string. Can have it take a `classNames` arg as input in case users want to customize it.

Reply:
would you like the same done for Assets/trees.tsx?

Reply:
Yup!

Reply:
`heroImage` now takes a style prop. 

I did the same for the `Trees` asset on the `CallToActionParagraphs`. I renamed this to `CtaAsset` as I saw it more fitting from the user's point of view. Let me know what you think.